src/osd/osd_types.cc

```diff
@@ -5091,6 +5100,7 @@ void object_info_t::dump(Formatter *f) const
   f->dump_unsigned("expected_write_size", expected_write_size);
   f->dump_unsigned("alloc_hint_flags", alloc_hint_flags);
   f->dump_object("manifest", manifest);
+  f->dump_stream("logical_extents") << extents;
```

Comment:
may as well make this structures.  open_array_section("extents"), open_object_section("extent"), dump_unsigned("offset", ...), ...